msal-dotnet-articles/acquiring-tokens/desktop-mobile/acquiring-tokens-interactively.md

```diff
@@ -71,84 +99,74 @@ Remarks:
-| WAM                                            | - FIDO, Hello, Conditional Access <br> - Fully integrated with Windows <br> - Better security <br> - This is the North Star of the Identity team; many experiences will light up with WAM use! <br> - See [WAM](./wam.md) for all benefits.             | - Legacy MSA-passthrough config does not work. We recommend creating a new app if to move away from MSA-passthrough. <br> - Windows only (10+, Server 2016 and Server 2019+). |
+## Browser support
+
+| Browser           | Pro                  | Con                |
+|:------------------|:---------------------|:-------------------|
+| Embedded WebView1 (based on Internet Explorer) | - Ships with all supported versions of Windows <br>- In use by identity libraries for 10+ years | - No FIDO support (e.g., YubiKey)<br>- No support for Windows Hello<br>- Conditional Access problems on older Windows versions<br>- Windows only |
+| Embedded WebView2 (based on Microsoft Edge)    | - FIDO and Windows Hello support | - Conditional Access problems on some older Windows versions.<br>- Windows only |
+| System browser                                 | - Uses default system browser.<br>- Chrome, Edge, and Firefox have integration with Conditional Access, Windows Hello, and FIDO.<br>- Works on macOS, Linux, and every possible version of Windows. | - Somewhat disruptive user experience (context switches to the browser). |
+| [Windows Broker](./wam.md)                     | - Support for FIDO, Windows Hello, and Conditional Access policies.<br>- Fully integrated with Windows.<br>- Better security.<br>- Long-term strategic component for authentication on Windows.<br> | - Legacy MSA-passthrough configuration does not work. We recommend creating a new app if you move away from MSA-passthrough.<br>- Windows only (10+, Server 2016, and Server 2019+). |
 
 ## How to
 
 ### Have the user consent upfront for several resources
 
 >[!NOTE]
-> Getting consent for several resources works for Azure AD v2.0, but not for Azure AD B2C. B2C supports only admin consent, not user consent.
+> Getting consent for several resources works for Microsoft Entra ID, but not for Microsoft Entra B2C. In the B2C scenario, only admin consent is supported.
 
-The Azure AD v2.0 endpoint does not allow you to get a token for several resources at once. Therefore the scopes parameter should only contain scopes for a single resource. However, you can ensure that the user pre-consents to several resources by using the `extraScopesToConsent` parameter.
+The Microsoft Entra ID endpoint does not allow you to get a token for several resources at once. The scopes parameter should only contain scopes for a single resource. However, developers can ensure that the user pre-consents to several resources by using the `extraScopesToConsent` argument.
```

Comment:
Delete this page since it's duplicate? https://learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/user-gets-consent-for-multiple-resources